src/Cart.java

```diff
@@ -0,0 +1,67 @@
+import classes.Product;
+
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.Objects;
+import java.util.Set;
+
+public class Cart {
+    private final Set<Product> products = new HashSet<Product>();
+    private Double totalPrice;
+
+    public Cart() {
+    }
+
+    public Set<Product> getProducts() {
+        return products;
+    }
+
+    public Double getTotalPrice() {
+        return totalPrice * 1.22;
```

Comment:
Quell 1.22 è l'iva ?

Reply:
Si, mi è sfuggito, avevo deciso di toglierlo. In realtà come abbiamo visto dai video non è un parametro che va calcolato lì

Reply:
tolto